From 79d6035ecfe678979aaf941acd6b9c3e3067619c Mon Sep 17 00:00:00 2001
From: Caesar Wang <wxt@rock-chips.com>
Date: Fri, 15 Jan 2016 21:49:51 +0800
Subject: [PATCH] UPSTREAM: ARM: dts: rockchip: enable the uart0 for kylin
 board

This patch is enabling the uart0 for bluetooth module.

Change-Id: Iad636b38df4c2a1ba4551bc104a343ab4e8aa7c3
Signed-off-by: Caesar Wang <wxt@rock-chips.com>
Signed-off-by: Heiko Stuebner <heiko@sntech.de>
(cherry picked from git.kernel.org next/linux-next.git master
 commit 5d69fa0f27f5bdff449858a20b7f50f08b9f33d8)
---
 arch/arm/boot/dts/rk3036-kylin.dts | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm/boot/dts/rk3036-kylin.dts b/arch/arm/boot/dts/rk3036-kylin.dts
index b28762522da0..c748d2cbeed6 100644
--- a/arch/arm/boot/dts/rk3036-kylin.dts
+++ b/arch/arm/boot/dts/rk3036-kylin.dts
@@ -308,6 +308,10 @@
 	pinctrl-0 = <&sdio_clk &sdio_cmd &sdio_bus4>;
 };
 
+&uart0 {
+	status = "okay";
+};
+
 &uart2 {
 	status = "okay";
 };
-- 
2.35.3

